### PR TITLE
Feat: added authentication

### PR DESCRIPTION
**Pull request by aiman-ansari**
Added Signup and login

- User can create an account
- User can login using existing account

### CODE REVIEW

File: src/Pages/Authentication/Login.js [hunk 1]
```diff
@@ -1,7 +1,75 @@
...
+    const navigate = useNavigate()
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const { handleLoginData} = useAuth()
+    const [error, setError] = useState(false)
+    const [message, setMessage] = useState('')
+    const test = {
+        email: "adarshbalika@gmail.com",
+        password: "adarshBalika123"
+    }
```
Comment (ShrutiLakhani) [inline src/Pages/Authentication/Login.js:18]: As Akanksha suggested, please use different credentials for your test user. Other than adarshbalika

**Review by MonikaJhingan — APPROVED** [review]
Looks good to be merged.

**Review by ShrutiLakhani — APPROVED** [review]
Code looks clean and good to merge. suggested one thing, you might want to change

File: src/Pages/Authentication/SignUp.js [hunk 2]
```diff
@@ -1,7 +1,84 @@
...
+            <div class="form">
+                <span className='bold-text'>Sign up</span>
+                <div class="input-with-icons ">
+                    <i class="bi bi-person-fill input-icon"></i>
+                    <input type="email" 
+                    placeholder="Enter your First Name" 
+                    class="icon-input"
+                    value={firstName}
+                    onChange={(e)=>setFirstName(e.target.value)}/>
+                </div>
```
Comment (rahulyadav139) [inline src/Pages/Authentication/SignUp.js:41]: Suggestions: here you are creating signup form, try to wrap inside a "form" html tag. Use semantic html.

**Review by rahulyadav139 — APPROVED** [review]
Every thing is looking great. Try to use semantic html. Other than that great work.

**Pull request merged**